STACpopulator/input.py

```diff
@@ -48,34 +44,6 @@ def __iter__(self) -> Iterator:
         """Iterate over this loader."""
         raise NotImplementedError
 
-    def reset(self) -> None:
-        """Reset the internal state of the generator."""
-        raise NotImplementedError
-
-
-class THREDDSCatalog(TDSCatalog):
```

Comment:
This was unused. The problem that this was trying to solve is implemented directly below.